library/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/monitor/DatadogRumMonitor.kt

```diff
@@ -93,6 +93,8 @@ internal class DatadogRumMonitor(
         handler.postDelayed(keepAliveRunnable, KEEP_ALIVE_MS)
     }
 
+    private val globalAttributes = mutableMapOf<String, Any?>()
```

Comment:
should we make it thread-safe?

Reply:
I think is lighter if we use a `ConcurrentHashMap` instead